client/src/components/common/PageBanner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import logo from '../../images/carrot_2.png';
+
+const PageBanner = (props) => {
```

Comment:
Actually don't know why it looks so strange here, but my editor is set to two spaces per tab and looks very different in my editor.

<img width="725" alt="image" src="https://user-images.githubusercontent.com/14282372/28807843-a684ea68-762c-11e7-8891-7a920966ba3b.png">